composer.lock

```diff
@@ -5824,6 +5825,7 @@
                         "[project-root]/Gruntfile.js": "assets/scaffold/files/Gruntfile.js",
                         "[project-root]/package.json": "assets/scaffold/files/package.json",
                         "[project-root]/package-lock.json": "assets/scaffold/files/package-lock.json",
+                        "[project-root]/modernizr-config.json": "assets/scaffold/files/modernizr-config.json",
```

Comment:
@escopecz I will add an extra github action check to ensure the `composer.json` and `composer.lock` 
 file are in sync. This change is unrelated to this PR, but can be confusing when doing QA

Reply:
Thanks!